v0/destinations/engage/data/EngageIdentifyConfig.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "sourceKeys": ["traits.email", "context.traits.email"],
+    "destKey": "email"
```

Comment:
is it possible to edit a user if email is not present? if not then lets keep email as a mandatory fields

Reply:
Yes it is possible to edit a user if email is not given

Reply:
but can we update the user who does not have email?

Reply:
ok